### ISSUE TITLE
accuracy issue

### ISSUE THREAD

**Issue by FrankCAN** [WangYueFt/dgcnn#12]
Hi,
Is there anyone who achieved about 92% result? Why I only got 89%? 

Many thanks.

Best Regards
Frank

**Comment by Haiyan-Chris-Wang** [comment]
How many epochs did you run?

**Comment by FrankCAN** [comment]
250 epochs as default.

**Comment by Haiyan-Chris-Wang** [comment]
I got the 92 result after I run the 250 epochs

**Comment by FrankCAN** [comment]
That's good, did you change any parameters in your code? As all the parameters I am using are default value. Many thanks for your help.

**Comment by Haiyan-Chris-Wang** [comment]
I didn't change. just as default

**Comment by FrankCAN** [comment]
Hi,
Can you please share your train.py file and your training log file?, Then I can check the difference between our paprameters,  many thanks.

**Comment by patrick-llgc** [comment]
Another data point. I got 90.99% accuracy after 250 epochs.

```
eval mean loss: 1.431454
eval accuracy: 0.909903
eval avg class acc: 0.884221
```

I looked back a few epochs, and the eval accuracy reaches about 91.4. When you say 89%, are you referring to the overall accuracy or avg class accuracy? @FrankCAN

**Comment by YimingCuiCuiCui** [comment]
Try to use Pytorch version rather than Tensorflow version and you will get a surprise.

**Comment by Yuquan-Lu** [comment]
> I got the 92 result after I run the 250 epochs

in paper,modelnet40 1024 points acc reach to 92.9%,2048 points reach to 93.5

**Comment by ShiQiu0419** [comment]
> Try to use Pytorch version rather than Tensorflow version and you will get a surprise.

Hi, I have not found any pytorch codes for spatial transform part. Did I miss them?

**Comment by grxiao** [comment]
i just got the result of 91.2
![image](https://user-images.githubusercontent.com/5366316/64323828-caee5700-cff7-11e9-9a5e-ac5c236569c4.png)

**Comment by grxiao** [comment]
i just got the result of 91.2%
![image](https://user-images.githubusercontent.com/5366316/64323894-f07b6080-cff7-11e9-8899-067f5cd4c4c9.png)
